prime-dotnet-webapi/prime.csproj

```diff
@@ -23,6 +23,11 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.0" />
     <PackageReference Include="QRCoder" Version="1.3.9" />
+    <PackageReference Include="RavenSharp.Core" Version="1.0.0-beta-1" />
+    <PackageReference Include="Sentry" Version="3.9.2" />
+    <PackageReference Include="Sentry.AspNetCore" Version="3.9.2" />
+    <PackageReference Include="Sentry.Serilog" Version="3.9.2" />
+    <PackageReference Include="System.Reflection.Metadata" Version="5.0.0" />
```

Comment:
Are `System.Reflection.Metadata` and `RavenSharp.Core` being used in the project?